vtkAddonInstallConfig.cmake.in

```diff
@@ -0,0 +1,6 @@
+set(vtkAddon_CMAKE_DIR "@CMAKE_INSTALL_PREFIX@/@vtkAddon_INSTALL_CMAKE_DIR@")
```

Comment:
Paths here should be relative to the location of the config file.

The following resources may be helpful to learn how to move forward:

* https://stackoverflow.com/questions/51564251/correct-way-to-use-third-party-libraries-in-cmake-project/51567322#51567322

* https://stackoverflow.com/questions/56135785/correctly-set-the-location-of-imported-cmake-targets-for-an-installed-package/56288628#56288628
   - and associated demo project: https://github.com/jcfr/stackoverflow-56135785-answer

Some CMake projects:

* proxTV: https://github.com/phcerdan/proxTV/blob/a503293ed567d7114bee85e4e068305d4afe7dbe/CMakeLists.txt#L84-L136

* python-cmake-buildsystem: https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/blob/2b036f110364a9295760345c9bbb3337c2ca9d72/CMakeLists.txt#L636-L687

* CTKAppLauncher: https://github.com/commontk/AppLauncher/blob/1c7156701ee96ab7b69ecc8322dc5443514927fa/CMakeLists.txt#L363-L469